alti/views/height.py

```diff
@@ -26,11 +26,21 @@ def __init__(self, request):
             self.layers = request.params.get('elevation_model')
         else:
             self.layers = ['DTM25']
+        if 'sr' in request.params:
+            self.sr = int(request.params.get('sr'))
+        else:
+            if (2.42e6 <= self.lon <= 2.9e6) and (1.03e6 <= self.lat <= 1.35e6):
+                self.sr = 2056
+            elif (0.42e6 <= self.lon <= 0.9e6) and (0.03e6 <= self.lat <= 0.35e6):
+                self.sr = 21781
+            else:
+                raise HTTPBadRequest("Your coordinates are neither in EPSG:21781 nor in EPSG:2056")
+
```

Comment:
Can't we try to use the srs guesser here? Otherwise, looks like duplicated code to me.